packages/calypso-products/src/sort.js

```diff
@@ -32,23 +33,26 @@ function sortProducts( products ) {
 
 	const includedItems = products.filter( isIncludedWithPlan );
 
-	domainItems = difference( products, includedItems );
+	domainItems = products.filter( ( product ) => ! includedItems.includes( product ) );
 	domainItems = domainItems.filter( isDomainProduct );
 	domainItems = Object.entries( groupBy( domainItems, 'meta' ) );
-	domainItems = sortBy( domainItems, function ( pair ) {
+	domainItems = domainItems.sort( function ( pair ) {
 		if ( pair[ 1 ][ 0 ] && pair[ 1 ][ 0 ].cost === 0 ) {
 			return -1;
 		}
 		return pair[ 0 ];
 	} );
```

Comment:
Replacing `sortBy` with `Array.prototype.sort` like this won't work. The callbacks are very different:
- `sortBy` callback takes an array item (usually object) as parameter and returns a value (primitive string or number) by which to sort.
- `sort` callback takes two array items as parameters and returns `-1`, `0` or `1` depending on the result of comparison it does.

Reply:
Also, beware that `sort` mutates the original array. In this case both `sort` calls are safe.

Reply:
Ah, thank you for catching that. I just saw that the function returned a `-1` sometimes and assumed it worked the same. In any case, this file has now been removed.